CoursesManager.MVVM/Messages/ToastNotificationMessage.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CoursesManager.UI.Messages;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CoursesManager.MVVM.Messages
+{
+    public class ToastNotificationMessage : BaseMessage<ToastNotificationMessage>
+    {
+        public bool SetVisibillity { get; set; }
+        public string NotificationText { get; set; }
+        public System.Enum ToastType { get; set; }
```

Comment:
Waarom System.Enum? ToastType was zelf toch ook een enum? Als je die gewoon als type gebruikt hoef je m in de mainwindowviewmodel ook niet te casten

Reply:
Veranderd